tools/dhtnode.cpp

```diff
@@ -277,15 +293,20 @@ void cmd_loop(std::shared_ptr<DhtRunner>& dht, std::map<std::string, dht::indexa
                 continue;
 
             indexation::Value v {h, 0};
-            indexes.at(index).insert(createPhtKey(parseStringMap(keystr)), v,
-                [=](bool success) {
-                    if (not success) {
-                        std::cout << "Pht::insert: failed." << std::endl;
-                        return;
+            try {
+                auto key = createPhtKey(parseStringMap(keystr));
+                indexes.at(index).insert(key, v,
+                    [=](bool success) {
+                        if (not success) {
```

Comment:
Should use same method as at line 280
Pht::insert: (success ) ? "done" : "failed"